README.md

```diff
@@ -1,5 +1,5 @@
 # MiniScaffold
-F# Template for creating and publishing libraries targeting .NET Full (net45) and Core (netstandard1.6)
+F# Template for creating and publishing libraries targeting .NET Full (net45) and Core (netstandard2.0)
```

Comment:
Why does the README mention `net45` here instead of `net461`? Oversight, and it should also be `net461`?